tutorials/shaders/shader_reference/shading_language.rst

```diff
@@ -816,20 +819,26 @@ used, and how the editor should allow users to modify it.
     uniform vec4 other_color : source_color = vec4(1.0); // Default values go after the hint.
     uniform sampler2D image : source_color;
 
-It's important to understand that textures *that are supplied as color* require
-hints for proper sRGB -> linear conversion (i.e. ``source_color``), as Godot's
-3D engine renders in linear color space. If this is not done, the texture will
-appear washed out.
+.. admonition:: Source Color
```

Comment:
This is the first use of an admonition note with a custom title in the whole docs repo. I think it makes this readable, but if this is against some style guide I can change this to use a subheader or standard note block instead.

Reply:
Looks good to me

Reply:
I'll be trying this out whenever a note block could use a title, then. I don't mind reverting such changes back if it proves to be a problem. I think in most cases, a note with a title should be a subsection, though?

Reply:
I think so. I suspect that most notes will be fine without a special title